[BUG] Compatibility issues with Foundry VTT v13

**Module Version:** v1.3.4

**Describe the bug**
The sidebar button for SimpleFog is no longer displayed after updating to the latest stable build of Foundry VTT v13. Pre-existing fog is rendered. 

**To Reproduce**
1. Create a new world using the D&D 5e ruleset
2. Enable the SimpleFog module and reset
3. Observe the missing control button
4. Observe following errors in the console
```
foundry.mjs:23613  Foundry VTT | Error thrown in hooked function '' for hook 'getSceneControlButtons'
#call @ foundry.mjs:23613
foundry.mjs:23633  Error: Error thrown in hooked function '' for hook 'getSceneControlButtons'. controls.push is not a function
    at Hooks.onError (foundry.mjs:23632:24)
    at #call (foundry.mjs:23614:36)
    at Hooks.callAll (foundry.mjs:23569:17)
    at #prepareControls (foundry.mjs:118024:13)
    at SceneControls._configureRenderOptions (foundry.mjs:117929:89)
    at #render (foundry.mjs:26953:10)
    at #try (foundry.mjs:7130:23)
    at foundry.mjs:7103:16
    at new Promise (<anonymous>)
    at Semaphore.add (foundry.mjs:7101:12)Caused by: TypeError: controls.push is not a function
    at Object.fn (simplefog.js:251:11)
    at #call (foundry.mjs:23610:20)
    at Hooks.callAll (foundry.mjs:23569:17)
    at #prepareControls (foundry.mjs:118024:13)
    at SceneControls._configureRenderOptions (foundry.mjs:117929:89)
    at #render (foundry.mjs:26953:10)
    at #try (foundry.mjs:7130:23)
    at foundry.mjs:7103:16
    at new Promise (<anonymous>)
    at Semaphore.add (foundry.mjs:7101:12)
onError @ foundry.mjs:23633
```

**Expected behavior**
The SimpleFog "cloud" button should be rendered in the sidebar on the left side of the screen. 

**Screenshots**
![Image](https://github.com/user-attachments/assets/a1321b7c-f181-4d7f-851d-4992fe33b37c)

**Browser:**
Microsoft Edge Version 135.0.3179.98 (Official build) (64-bit)

**Foundry Version:** 
Foundry VTT v13.342

**Game System:** 
D&D 5e

**Additional context**
Pre-existing fog seems to work. I haven't tested whether the hotkey still works as I don't know which key combination it uses.

--- a/src/module/apps/BrushControls.js
+++ b/src/module/apps/BrushControls.js
@@ -52,8 +52,8 @@ export class BrushControls extends HandlebarsApplicationMixin(ApplicationV2) {
 	/** @inheritDoc */
 	_onFirstRender(context, options) {
 		super._onFirstRender(context, options);
-		const left = ui.nav?.element[0].getBoundingClientRect().left;
-		const top = ui.controls?.element[0].getBoundingClientRect().top;
+		const left = ui.nav?.position.left;
+		const top = ui.controls?.position.top;
 		options.position = {...options.position, left, top};
 		canvas.scene.apps[this.id] = this;
 	}

--- a/src/module/simplefog.js
+++ b/src/module/simplefog.js
@@ -160,8 +160,8 @@ Hooks.on("canvasInit", () => {
  */
 Hooks.on("getSceneControlButtons", (controls) => {
 	if (!game.user.isGM) return;
-	const tools = [
-		{
+	const tools = {
+		"simplefogtoggle": {
 			name: "simplefogtoggle",
 			title: game.i18n.localize("SIMPLEFOG.onoff"),
 			icon: "fas fa-eye",
@@ -169,37 +169,37 @@ Hooks.on("getSceneControlButtons", (controls) => {
 			active: canvas.simplefog?.visible,
 			toggle: true,
 		},
-		{
+		"brush": {
 			name: "brush",
 			title: game.i18n.localize("SIMPLEFOG.brushTool"),
 			icon: "fas fa-paint-brush",
 			onClick: () => canvas.simplefog?._changeTool(),
 		},
-		{
+		"room": {
 			name: "room",
 			title: game.i18n.localize("SIMPLEFOG.roomTool"),
 			icon: "fas fa-block-brick",
 			onClick: () => canvas.simplefog?._changeTool(),
 		},
-		{
+		"polygon": {
 			name: "polygon",
 			title: game.i18n.localize("SIMPLEFOG.polygonTool"),
 			icon: "fas fa-draw-polygon",
 			onClick: () => canvas.simplefog?._changeTool(),
 		},
-		{
+		"box": {
 			name: "box",
 			title: game.i18n.localize("SIMPLEFOG.boxTool"),
 			icon: "far fa-square",
 			onClick: () => canvas.simplefog?._changeTool(),
 		},
-		{
+		"ellipse": {
 			name: "ellipse",
 			title: game.i18n.localize("SIMPLEFOG.ellipseTool"),
 			icon: "far fa-circle",
 			onClick: () => canvas.simplefog?._changeTool(),
 		},
-		{
+		"sceneConfig": {
 			name: "sceneConfig",
 			title: game.i18n.localize("SIMPLEFOG.sceneConfig"),
 			icon: "fas fa-cog",
@@ -208,7 +208,7 @@ Hooks.on("getSceneControlButtons", (controls) => {
 			},
 			button: true,
 		},
-		{
+		"clearfog": {
 			name: "clearfog",
 			title: game.i18n.localize("SIMPLEFOG.reset"),
 			icon: "fas fa-trash",
@@ -238,7 +238,7 @@ Hooks.on("getSceneControlButtons", (controls) => {
 			},
 			button: true,
 		},
-	];
+	};
 	let activeTool = game.settings.get("simplefog", "toolHotKeys");
 	if (canvas.grid?.type) {
 		tools.splice(2, 0, {

--- a/src/module/simplefog.js
+++ b/src/module/simplefog.js
@@ -248,14 +248,14 @@ Hooks.on("getSceneControlButtons", (controls) => {
 			onClick: () => canvas.simplefog?._changeTool(),
 		});
 	} else if (activeTool === "grid") activeTool = "brush";
-	controls.push({
+	controls["simplefog"] = {
 		name: "simplefog",
 		title: game.i18n.localize("SIMPLEFOG.sf"),
 		icon: "fas fa-cloud",
 		layer: "simplefog",
 		tools,
 		activeTool,
-	});
+	};
 });
 
 /**

